.Xresources

```diff
@@ -11,6 +11,9 @@ st.borderless:        0
 ! --- The following options options can be reloaded via USR1 signal. ---
 st.font:              Liberation Mono:pixelsize=12:antialias=true:autohint=true;
 
+! --- Font2 fallback, each font can be separated by comma ",". ---
+st.font_fallback:     Fira Code Nerd Font Mono:style=Regular:pixelsize=13:antialias=true,Material\\-Design\\-Iconic\\-Font:style=Design-Iconic-Font:pixelsize=13,JoyPixels:pixelsize=13
```

Comment:
This is meant to show the default value -- add a comment noting it's blank by default -- also do you need to escape the `-` here in the name?

Reply:
Yep, you need to escape if the font name actually contains `\-`. But not sure what you mean by adding a comment noting it's blank by default?

![image](https://user-images.githubusercontent.com/51831435/142055447-e60d1004-f2cf-4b4d-9340-8c610634a288.png)
![image](https://user-images.githubusercontent.com/51831435/142055503-e9ce1dac-c018-4ec3-80c2-bcbe46acac36.png)

Reply:
> But not sure what you mean by adding a comment noting it's blank by default.

Meaning, if the user has an empty .Xresources, the font_fallback value is blank